stdlib/LinearAlgebra/src/lapack.jl

```diff
@@ -95,6 +95,23 @@ function chkfinite(A::AbstractMatrix)
     return true
 end
 
+function chkuplofinite(A::AbstractMatrix, uplo::AbstractChar)
+    m, n = size(A)
+    if uplo == 'U'
+        for i in 1:m, j in i:n
```

Comment:
what do you think about changing the loops to iterate column-wise?
` for j = 1:n, i = 1:j`